src/components/SliderBox/components/SliderArrow/SliderArrow.scss

```diff
@@ -0,0 +1,35 @@
+.slider-card-button-prev-arrow,
+.slider-card-button-next-arrow {
+  display: none;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  position: absolute;
+  transform: translateY(-50%);
+  opacity: 0.5;
+  top: 50%;
```

Comment:
please find another approach for displaying arrows

Reply:
fixed